exo-run/src/docker-runner.ls

```diff
@@ -63,11 +65,20 @@ class DockerRunner extends EventEmitter
         @emit 'online'
 
 
-  _check-dependency-containers: ~>
-    for dependency of @docker-config.dependencies
-      app-dependency = "#{@docker-config.app-name}-#{dependency}"
-      DockerHelper.ensure-container-is-running app-dependency, dependency
-      @docker-config.env[dependency.to-upper-case!] = DockerHelper.get-docker-ip app-dependency
+  _check-dependency-containers: (done) ~>
+    dependencies = []
+    for image, vars of @docker-config.dependencies
```

Comment:
Let's use correct variable names here: `dependency-name` and `dependency-config` or something similar.